docs/orchestrating-elastic-stack-applications/enterprise-search.asciidoc

```diff
@@ -0,0 +1,242 @@
+:page_id: enterprise-search
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Run Enterprise Search on ECK
+
+This section describes how to deploy, configure and access Enterprise Search with ECK.
+
+* <<{p}-enterprise-search-quickstart,Quickstart>>
+* <<{p}-enterprise-search-configuration,Configuration>>
+
+[id="{p}-enterprise-search-quickstart"]
+== Quickstart
+
+. Apply the following specification to deploy Enterprise Search. ECK automatically configures the secured connection
+to an Elasticsearch cluster named `quickstart`, created in link:../k8s-quickstart.html[Elasticsearch quickstart].
++
+[source,yaml,subs="attributes,+macros"]
+----
+cat $$<<$$EOF | kubectl apply -f -
+apiVersion: enterprisesearch.k8s.elastic.co/v1beta1
+kind: EnterpriseSearch
+metadata:
+  name: enterprise-search-quickstart
+spec:
+  version: {version}
+  count: 1
+  elasticsearchRef:
+    name: quickstart
+EOF
+----
+
+. Monitor the Enterprise Search deployment.
++
+Retrieve details about the Enterprise Search deployment:
++
+[source,sh]
+----
+kubectl get enterprisesearch
+----
++
+[source,sh,subs="attributes"]
+----
+NAME                            HEALTH    NODES    VERSION   AGE
+enterprise-search-quickstart    green     1        {version}      8m
+----
++
+List all the Pods belonging to a given deployment:
++
+[source,sh]
+----
+kubectl get pods --selector='enterprisesearch.k8s.elastic.co/name=enterprise-search-quickstart'
+----
++
+[source,sh]
+----
+NAME                                                READY   STATUS    RESTARTS   AGE
+enterprise-search-quickstart-ent-58b84db85-dl7c6    1/1     Running   0          2m50s
+----
++
+. Access logs for that Pod:
++
+[source,sh]
+----
+kubectl logs -f enterprise-search-quickstart-ent-58b84db85-dl7c6
+----
+
+. Access Enterprise Search
++
+A ClusterIP service is automatically created for the deployment,
+and can be used to access the Enterprise Search API from within the Kubernetes cluster:
++
+[source,sh]
+----
+kubectl get service enterprise-search-quickstart-ent-http
+----
++
+Use `kubectl-portforward` to access Enterprise Search from your local workstation:
++
+[source,sh]
+----
+kubectl port-forward service/enterprise-search-quickstart-ent-http 3002
+----
++
+Open `https://localhost:3002` in your browser.
++
+NOTE: Your browser will show a warning because the self-signed certificate configured by default is not verified by a third party certificate authority and not trusted by your browser. Acknowledge the warning for the purposes of this quickstart but it is highly recommended that you <<{p}-enterprise-search-expose,configure valid certificates>> for any production deployments.
```

Comment:
```suggestion
NOTE: Your browser will show a warning because the self-signed certificate configured by default is not verified by a self-signed certificate authority and not trusted by your browser. Acknowledge the warning for the purposes of this quickstart but it is highly recommended that you <<{p}-enterprise-search-expose,configure valid certificates>> for any production deployments.
```

Reply:
uh?

Reply:
ah I got what you mean.
Let's maybe change it to `not verified by a known certificate authority`?

Reply:
👍